### DISCUSSION TITLE
Wiki.js installed in the intranet environment, locale configuration cannot be synchronized

### DISCUSSION THREAD

**Discussion by MrKervin** [requarks/wiki#3142] [Help / Questions] [answered]
Our company computers are prohibited from accessing external networks. I try to build wiki.js on a server that is forbidden to access the Internet, but the locale configuration needs to be connected to the Internet to be able to synchronize.Is there any good way to solve it?For example, directly modify the locale default settings.

**Comment by mattpr** [comment]
Howdy.  Possible you didn't know quite what to look for.

I didn't find anything on the locales page, but on `/dev/translations` page (https://docs.requarks.io/dev/translations)...

> If you're using offline sideloading, locale files are updated every day at 12:00 AM Eastern Standard Time (EST).

Which led me to this:

https://docs.requarks.io/install/sideload

> If your wiki is installed in an environment which is isolated from the internet, you can sideload data that would normally be downloaded from the internet.
> This is achieved by manually downloading a set of files and placing them in a specific directory in your wiki installation. These files will be imported during initialization.
> In order to install locale packages, you need...
> Run Wiki.js (or restart the process if already running) to automatically sideload the files localed in the data/sideload folder.

Might be useful to have link/reference to sideload page on the relevant config pages (ie themes, locales, etc).

**Comment by MrKervin** [comment]
![image](https://user-images.githubusercontent.com/74222081/109090275-afc45980-774d-11eb-808c-064e08bf3a76.png)
First of all, thank you very much for your help, but after I tried the above method, the problem is still not solved, is it because I did something wrong.

**Comment by MrKervin** [comment, marked as answer]
The problem is solved. The cause of the problem is that there is no offline mode configuration in config.yml in the image package, and ‘offline: true’ needs to be manually added.
![image](https://user-images.githubusercontent.com/74222081/109132896-50873900-778f-11eb-8a4a-567a6bef2d8e.png)

**Comment by mattpr** [comment]
Great.  Glad you got it figured out.  You might want to mark your comment as solving/answering this question so other people know this is no longer an open issue.

![image](https://user-images.githubusercontent.com/1196252/109275387-f466f800-7814-11eb-9a35-de7aebaa0a4b.png)
